app/helper/StringHelper.scala

```diff
@@ -51,4 +53,8 @@ object StringHelper {
   def capitalizeName(name: String): String =
     """[\P{P}&&\P{Z}]+""".r.replaceAllIn(name, m => m.group(0).toLowerCase.capitalize)
 
+  final object TrimedString {
```

Comment:
plutôt
```suggestion
  final object NonEmptyTrimmedString {
```
c'est plus facile à lire hors contexte ?

Reply:
Oui je suis d'accord